src/essentials/Colors/types.ts

```diff
@@ -4,6 +4,58 @@ export type HSL = `hsl(${number}, ${number}%, ${number}%)` | `hsla(${number}, ${
 
 export type Color = HSL | 'transparent';
 
+export type BareColorsSchema = {
```

Comment:
Wait, this is incorrect. There is no bare schema. The object differ between different themes. For example, Modern schema has one blue color and two types of red

Reply:
Yeah, I understand now, the classic and modern colors structures differ, so we can't have a common schema. I'm thinking on what the best approach is then, because I need a replacement type for what was before `Colors` and is now the bare colors css variables (i.e. `'var(--wave-b-color-white)' | 'var(--wave-b-color-black)' | 'var(--wave-b-color-blue-primary-1100)' | ...`)

I could rename `BareColorsSchema` to `ClassicBareColorsSchema` to make it clear that it's not a common schema for both modern and classic (for modern I don't need it at all). Wdyt? Do you see other possible solutions?

Reply:
I've removed this in the end